theme.json

```diff
@@ -1051,6 +1051,67 @@
 					"fontSize": "var:preset|font-size|x-large"
 				}
 			},
+			"core/quote": {
```

Comment:
Can we please rebase, and check how these styles live together with the work already done in the `theme.json`? There may be some that can be redundant after that.